docs/src/content/docs/fr/index.mdx

```diff
@@ -39,19 +41,122 @@ import AboutAstro from '~/components/about-astro.astro';
 		Apportez votre langage de balisage préféré. Starlight vous offre une
 		validation frontmatter intégrée avec la sécurité de type TypeScript.
 	</Card>
-	<Card
-		title="Apportez vos propres composants d'interface utilisateur"
-		icon="puzzle"
-	>
+	<Card	title="Apportez vos propres composants d'interface utilisateur" icon="puzzle">
 		Starlight est une solution de documentation complète, indépendante du cadre
 		de travail. Étendez avec React, Vue, Svelte, Solid, et plus encore.
 	</Card>
 </CardGrid>
 
-<AboutAstro title="Présenté par">
-Astro est le framework web tout-en-un conçu pour la vitesse.
-Tirez votre contenu de n'importe où et déployez-le partout, le tout alimenté par vos composants et bibliothèques d'interface utilisateur préférés.
+<TestimonialGrid title="What people are saying">
+	<Testimonial
+		name="Rachel"
+		handle="rachelnabors"
+		cite="https://twitter.com/astrodotbuild/status/1724934718745915558"
+	>
+		L'équipe d'Astro a ÉVOLUÉ la façon dont les documents peuvent être réalisés et vous pouvez tout obtenir dès le départ avec leur projet Starlight.
```

Comment:
```suggestion
		L'équipe d'Astro a fait ÉVOLUER la façon dont les documentations peuvent être réalisées et vous pouvez tout obtenir immédiatement avec leur projet Starlight.
```

A few tweaks which I think make it more correct for the first change and sound better for the last one.